Library/Formula/exim.rb

```diff
@@ -1,11 +1,9 @@
-require "formula"
-
 class Exim < Formula
   desc "Complete replacement for sendmail"
   homepage "http://exim.org"
-  url "http://ftp.exim.org/pub/exim/exim4/old/exim-4.85.tar.bz2"
-  mirror "https://www.mirrorservice.org/sites/ftp.exim.org/pub/exim/exim4/old/exim-4.85.tar.bz2"
-  sha256 "13211f2bbc5400d095a9b4be075eb1347e0d98676fdfe4be8a3b4d56281daaa4"
+  url "http://ftp.exim.org/pub/exim/exim4/exim-4.86.tar.gz"
+  mirror "https://github.com/Exim/exim/archive/exim-4_86.tar.gz"
```

Comment:
SHA mismatch on the mirror again: `812b9eab2afb32a60a4f01b5dc4fdc49ae77473d130bf9e18c42ea73e5db2aa8`.  (FYI, this is very common with GitHub archive tarballs, since they're just a snapshot of the repository.)